tutorials/algorithms/10_pvqd.ipynb

```diff
@@ -0,0 +1,591 @@
+{
```

Comment:
<p>I believe you can avoid this warning if you explicitly take the real part of the exact magnetization. As it is now, the <code>exact_magnetization</code> is a list of <code>complex</code> numbers with imaginary part 0.</p><p> </p><p>Also when I run the code as it is now I got the error <code>module 'scipy' has no attribute 'linalg'</code> . I had to add <code>import scipy.linalg</code> for it to work. Might be a problem with my version</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-tutorials/pull/1382/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>good point, I added the <code>.real</code> ! Yeah scipy is a bit weird sometimes depending on the local installation... </p>